randovania/generator/pickup_pool/pool_creator.py

```diff
@@ -55,43 +55,43 @@ def calculate_pool_results(layout_configuration: BaseConfiguration,
     return base_results
 
 
-def calculate_pool_pickup_count(layout: BaseConfiguration) -> tuple[int, int]:
+def calculate_pool_pickup_count(layout: BaseConfiguration) -> dict[LocationCategory | str, tuple[int, int]]:
```

Comment:
I think maybe this should be a simple `NamedTuple`. it's not obvious anywhere that index 0 is pickups and index 1 is locations